pagopa/io-app
# feat(Sicilia Vola): [IASV-32] Summary screen

## Short description
This PR add the SummaryScreen which allows the user to see the choices previously selected and the list of available destinations.

## List of changes proposed in this pull request
- Removed `getRegioni` request and associated store section (no longer used)
- Removed `getProvince` request and associated store section (no longer used)
- Completed the `SummaryScreen`
- Substituted the new `GetAeroportiStato` and `GetAeroportiBeneficiario` clients with the new `GetAeroportiAmmessi`
- Updated the `Municipality` type according to the new API


https://user-images.githubusercontent.com/11773070/137107667-29811761-1277-4b52-84a6-28661785920d.mp4


https://user-images.githubusercontent.com/11773070/137107678-a46034fe-ea12-4f52-8a14-e4ac588b270d.mp4



## How to test
Starts the flow from the SiciliaVola service detail press the Voucher Generation button.
Complete the flow.

If you choose a category different from "Disabled" dispatch the appropriate action in the selectDestination screen.
For example if you choose the "Student" dispatch in `ts/features/bonus/siciliaVola/screens/voucherGeneration/StudentSelectDestinationScreen.tsx` this action:
```
props.selectUniversity({
        state: { id: 1, name: "Italy" },
        municipality: {
          id: "ITA01",
          name: "Milan",
          longitude: 120,
          latitude: 120
        },
        universityName: "UniversityName"
      })

const mapDispatchToProps = (dispatch: Dispatch) => ({
...
  selectUniversity: (university: University) =>
    dispatch(svGenerateVoucherSelectUniversity(university))
}
```

Make sure to use this branch on the dev server [IASV-32-update-available-destination](https://github.com/pagopa/io-dev-api-server/tree/IASV-32-update-available-destination)
